lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -446,7 +445,12 @@ namespace Js
             // Clear disable implicit call bit as initialization code doesn't have any side effect
             Js::ImplicitCallFlags saveImplicitCallFlags = scriptContext->GetThreadContext()->GetImplicitCallFlags();
             scriptContext->GetThreadContext()->ClearDisableImplicitFlags();
-            JavascriptFunction::CallRootFunctionInScript(function, Js::Arguments(callInfo, args));
+
+            {
+                AutoSuspendDebuggerRecording autoSuspendDebuggerRecording(scriptContext);
```

Comment:
AutoProfilingUserCode is related to ThreadContext and AutoSuspendDebuggerRecording is in ScriptContext. Having trouble with types in compilation.